rosgraph_monitor/package.xml

```diff
@@ -10,8 +10,8 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
   <buildtool_depend>generate_parameter_library</buildtool_depend>
 
-  <depend>diagnostic_aggregator</depend>
   <depend>diagnostic_msgs</depend>
+  <depend>diagnostic_updater</depend>
```

Comment:
add diagnostic_aggregator as exec_ or test_ depend? for the example or a provided launchfile